pages/westmint/u/_id.vue

```diff
@@ -139,12 +139,13 @@ import isShareMode from '@/utils/isShareMode'
 import shouldUpdate from '@/utils/shouldUpdate'
 import shortAddress from '@/utils/shortAddress'
 import collectionList from '@/queries/unique/collectionListByAccount.graphql'
-import nftListByIssuer from '@/queries/unique/nftListByIssuer.graphql'
-import nftListCollected from '@/queries/unique/nftListCollected.graphql'
-import nftListSold from '@/queries/unique/nftListSold.graphql'
-import firstNftByIssuer from '@/queries/unique/firstNftByIssuer.graphql'
 import PrefixMixin from '@/utils/mixins/prefixMixin'
 
```

Comment:
[:)](https://github.com/kodadot/nft-gallery/issues/2918#issuecomment-1120378565)

Reply:
Oh, I see, you are saying be exactly like `'@/queries/subsquid/general/firstNftByIssuer.graphql'`